### ISSUE TITLE
about Pretrained word vector file

### ISSUE THREAD

**Issue by zl-1516** [monologg/NER-Multimodal-pytorch#2]
After downloading pretrained word embedding, there are three files in the word2vec folder, word2vec_200dim.model, word2vec_200dim.model.syn0.npy, word2vec_200dim.model.syn1neg.npy, 
 I want know how to use these three files as word_vector_200d.vec in the code.
thank you very much  !

**Comment by monologg** [comment]
@zl-1516 
I've used three files using `gensim` library. But it takes quite a long time to load and convert the vector. So I've compressed it into `word_vector_200d.vec` and uploaded on google drive. It will automatically downloaded when you run the code.

Plz check [this code](https://github.com/monologg/NER-Multimodal-pytorch/blob/master/utils.py#L25-L27)

This is the [link](https://drive.google.com/open?id=1jfIztxGXQJLCYHxSPBIOaSDeySvevti-) to download the file directly
